plugins/system/collect_linux.go

```diff
@@ -58,7 +62,8 @@ func (s *System) collect() {
 	}
 
 	if mem_total != 0.0 {
-		s.updaters[memUsageGauge].Update(mem_free / mem_total * 100)
+		avaialbe := (mem_free + buffers + cached)
```

Comment:
The commit message should contain the bug number so that it cross references to jira.